sdcm/cluster.py

```diff
@@ -5879,6 +5879,24 @@ def save_sct_dashboards_config(self, node):
         node.remoter.run('mkdir -p {}'.format(sct_monitoring_addons_dir), ignore_status=True)
         node.remoter.send_files(src=self.sct_dashboard_json_file, dst=sct_monitoring_addons_dir)
 
+    def update_default_time_range(self, start_timestamp: float, end_timestamp: float) -> None:
+        """
+        Specify the Grafana time range for all dashboards by updating the JSON files.
+
+        This method will find all JSON files in the `grafana/build/<subdir>` directories and replace
+        the "from" and "to" time range values with the provided start and end timestamps.
+        """
+        start_iso = datetime.fromtimestamp(start_timestamp, tz=timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ')
+        end_iso = datetime.fromtimestamp(end_timestamp, tz=timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ')
+        cmd = (f"find {os.path.join(self.monitor_install_path, 'grafana/build/')} -name '*.json' -exec "
```

Comment:
Two questions:

Wouldn't it be better to parse the dashboard json and change the fields? I believe the `from` field is in the top level of the dashboard object.

Also, does this need a reload or does it persist after reload? That is, how does grafana handle this change?

Reply:
in `grafana/build/` dir there are multiple dashboard files, different for each scylladb release. This command changes `from` and `to` fields in each of it (there's only one of them in each file).
This becomes the default.